staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -4208,8 +4208,8 @@ type ServiceSpec struct {
 
 	// If specified and supported by the platform, this will restrict traffic through the cloud-provider
 	// load-balancer will be restricted to the specified client IPs. This field will be ignored if the
-	// cloud-provider does not support the feature."
-	// More info: https://kubernetes.io/docs/tasks/access-application-cluster/create-external-load-balancer/
```

Comment:
this link is correct, it points to the create-external-load-balancer page and it works for me